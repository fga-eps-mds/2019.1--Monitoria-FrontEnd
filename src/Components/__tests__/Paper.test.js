import React from 'react';
import Paper from '../Feed/Paper';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Testing Paper component', () => {
    it('Test if Paper renders correctly', () =>{
        const tree = shallow(<Paper/>);
        expect(toJson(tree)).toMatchSnapshot();
    });
});
