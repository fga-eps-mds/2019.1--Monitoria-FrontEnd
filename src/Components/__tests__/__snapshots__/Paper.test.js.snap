// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Paper component Test if Paper renders correctly 1`] = `
<RecipeReviewCard
  classes={
    Object {
      "actions": "RecipeReviewCard-actions-3",
      "avatar": "RecipeReviewCard-avatar-6",
      "card": "RecipeReviewCard-card-1",
      "expand": "RecipeReviewCard-expand-4",
      "expandOpen": "RecipeReviewCard-expandOpen-5",
      "media": "RecipeReviewCard-media-2",
    }
  }
/>
`;
